packages/vm/src/calls.rs

```diff
@@ -232,6 +232,35 @@ mod tests {
             .unwrap();
     }
 
+    #[test]
+    fn call_migrate_works() {
+        let mut instance = mock_instance(&CONTRACT, &[]);
+
+        // init
+        let info = mock_info("creator", &coins(1000, "earth"));
+        let msg = r#"{"verifier": "verifies", "beneficiary": "benefits"}"#.as_bytes();
+        call_init::<_, _, _, Empty>(&mut instance, &mock_env(), &info, msg)
+            .unwrap()
+            .unwrap();
+
+        // change the verifier via migrate
+        let new_verifier = HumanAddr::from("someone else");
+        let msg = MigrateMsg {
+            verifier: new_verifier,
+        };
+        let info = mock_info(creator.as_str(), &[]);
+        let res = call_migrate(&mut instance, &mock_env(), &info, msg);
```

Comment:
We need a different `call_migrate` here than in the integration test. The integration test uses https://github.com/CosmWasm/cosmwasm/blob/master/packages/vm/src/testing/calls.rs (i.e. from the `testing::` module) but here we use a similar but different function from https://github.com/CosmWasm/cosmwasm/blob/master/packages/vm/src/calls.rs. This does not require the `MigrateMsg` type. Instead pass the migrate message as JSON.